### ISSUE TITLE
Please add fix to properly use own trained 512 model

### ISSUE THREAD

**Issue by netrunner-exe** [neuralchen/SimSwap#246]
Now option 512 is set to use beta 512 only.  I hope this will remove a lot of future questions about using own 512 model. If I understood correctly, my suggestion:
In `test_video....py` and` test_wholeimage...py` change

```
    if crop_size == 512:
        opt.which_epoch = 550000
        opt.name = '512'
        mode = 'ffhq'
    else:
        mode = 'None'
```
to

```
    if crop_size == 512:
        mode = 'ffhq'
    else:
        mode = 'None'

    if opt.name == '512':
        opt.which_epoch = 550000
```
And in [Inference for image or video face swapping](https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/usage.md) in example about 512:

`--name people` to `--name 512`

Thanks guys, you are the best! We are all looking forward to your new paper and SimSwap ++! Thanks!

**Comment by papipulato** [comment]
are you have 512 model now sir ? how much epoch do you train ?

**Comment by netrunner-exe** [comment]
> are you have 512 model now sir ? how much epoch do you train ?

No, just try to train 512.  I have a weak GPU, and in Colab it still takes a very long time to do this. Now I'm trying to make hq dataset before train 224
